### ISSUE TITLE
Runtime crash

### ISSUE THREAD

**Issue by bre7** [Zewo/DockerExamples#2]
Is this Docker image working ? When trying to run it, the following error appears:

> Todo: /home/buildslave/jenkins/workspace/oss-swift-linux-packages-ubuntu_14_04-one-off-build/swift/include/swift/Runtime/../../../stdlib/public/SwiftShims/RefCount.h:207: bool StrongRefCount::doDecrementShouldDeallocate() [ClearPinnedFlag = false]: Assertion `newval + quantum >= RC_ONE && "releasing reference with a refcount of zero"' failed.
> Aborted

Same as https://bugs.swift.org/browse/SR-175 ?

**Comment by jimrutherford** [comment]
I've just started working with the Docker container here and have found a few issues.  I've forked the repo (https://github.com/jimrutherford/DockerExamples) and am making changes and PR's as I come across issues.

The current state is that I have the project building but its failing now on Postgres (likely due to the fact that this container does not yet have a postgres instance setup).  This will be my next step to fix.

Feel free to clone my fork and contribute if you have the time?
